apisix/plugins/limit-count/limit-count-redis-cluster.lua

```diff
@@ -32,7 +32,7 @@ local mt = {
 
 local function new_redis_cluster(conf)
     local config = {
-        name = "apisix-redis-cluster",
+        name = "apisix-redis-cluster-" .. conf.name,
```

Comment:
I am confused by the solution. What is the relation between the password and name?

Reply:
one name Mapping one routes，if all redis_cluster config the same name. When concurrent lrucache can not find the correct redis_cluster in pool. 
This is my personal opinion ，maybe it is wrong.😄

Reply:
I read through the `resty.rediscluster`. Look like this library uses a module-level `slot_cache`, and the key is the `config.name`.

So it is no doubted that different configurations with the same name will collide.

What about using `"apisix-redis-cluster-" .. tostring(conf)` directly?
Better to add a comment about this change as future maintainer may not read through the library.